tests/online_ddl/run.sh

```diff
@@ -43,58 +46,67 @@ function real_run() {
 		"\"result\": true" 1
 
 	# imitate a DM task is started during the running of online DDL tool
-	if [ "$online_ddl_scheme" == "gh-ost" ]; then
-		run_sql_source1 "create table online_ddl.ignore (c int); create table online_ddl._ignore_gho (c int);"
-	elif [ "$online_ddl_scheme" == "pt" ]; then
-		run_sql_source1 "create table online_ddl.ignore (c int); create table online_ddl._ignore_new (c int);"
-	fi
+	run_sql_source1 "create table online_ddl.gho_ignore (c int); create table online_ddl._gho_ignore_gho (c int);"
+	run_sql_source1 "create table online_ddl.pt_ignore (c int); create table online_ddl._pt_ignore_new (c int);"
 
 	# start DM task only
-	cp $cur/conf/dm-task.yaml $WORK_DIR/dm-task-${online_ddl_scheme}.yaml
-	sed -i "s/online-ddl-scheme-placeholder/${online_ddl_scheme}/g" $WORK_DIR/dm-task-${online_ddl_scheme}.yaml
-	dmctl_start_task "$WORK_DIR/dm-task-${online_ddl_scheme}.yaml" "--remove-meta"
+	cp $cur/conf/dm-task.yaml $WORK_DIR/dm-task.yaml
+	dmctl_start_task "$WORK_DIR/dm-task.yaml" "--remove-meta"
 
 	echo "use sync_diff_inspector to check full dump data"
```

Comment:
After the start task here, tidb has no data, I don’t know which configuration file is written wrong, PTAL @glorv @lance6716

Reply:
maybe use `dmctl query-status` to see task status? or you can check dm-worker's log file

Reply:
I found that dm-task.yaml did not know how it was deleted by me when I mentioned pr. Let me change it first....

Reply:
An error log is as follow:

```log

[2021/07/22 12:41:14.212 +08:00] [ERROR] [subtask.go:311] ["unit process error"] [subtask=test] [unit=Load] ["error information"="{\"ErrCode\":34004,\"ErrClass\":\"load-unit\",\"ErrScope\":\"internal\",\"ErrLevel\":\"high\",\"Message\":\"restore data file (online_ddl.t2_pt.0000000000000.sql) failed: file dumped_data.test/online_ddl.t2_pt.0000000000000.sql: mapping row data [1 20001 José Arcadio Buendía NULL] for table \\u0026{sourceSchema:online_ddl sourceTable:t2_pt targetSchema:online_ddl targetTable:t_pt_target columnNameList:[id uid name info] insertHeadStmt:INSERT INTO `t_pt_target` VALUES}\",\"RawCause\":\"the suffix of 2_pt can't be converted to int64 not valid\"}"]
[2021/07/22 12:41:14.213 +08:00] [ERROR] [subtask.go:311] ["unit process error"] [subtask=test] [unit=Load] ["error information"="{\"ErrCode\":34004,\"ErrClass\":\"load-unit\",\"ErrScope\":\"internal\",\"ErrLevel\":\"high\",\"Message\":\"restore data file (online_ddl.t1_pt.0000000000000.sql) failed: file dumped_data.test/online_ddl.t1_pt.0000000000000.sql: mapping row data [1 10001 Gabriel García Márquez NULL] for table \\u0026{sourceSchema:online_ddl sourceTable:t1_pt targetSchema:online_ddl targetTable:t_pt_target columnNameList:[id uid name info] insertHeadStmt:INSERT INTO `t_pt_target` VALUES}\",\"RawCause\":\"the suffix of 1_pt can't be converted to int64 not valid\"}"]
```

Seems that I should alter the table name to meet the requirements of the convert task?

Reply:
seems you configurated column mapping wrongly. We can skip checking it now.

Reply:
It's because the third argument of `arguments` should be the same as the prefix of `table-pattern`.

```yaml
column-mappings:
  gho-instance-1:
    schema-pattern: "online_ddl"
    table-pattern: "gho_t*"
    expression: "partition id"
    source-column: "id"
    target-column: "id"
    arguments: ["1", "", "gho_t"]
```
> https://github.com/pingcap/dm/pull/1904/files#diff-13484eff718862326f14f11f7fefeb5b0cc8d2688932e779de36e77c3da6d39bR61-R68